### ISSUE TITLE
 rails g groq:install doesn't work

### ISSUE THREAD

**Issue by aka-nez** [drnic/groq-ruby#11]
```
-> rails g groq:install
Could not find generator 'groq:install'. (Rails::Command::CorrectableNameError)
Run `bin/rails generate --help` for more options.
```

Rails 7.1.3, ruby 3.2.0

**Comment by aka-nez** [comment]
Thanks for this gem! I was able to replace Replicate with Groq in 30 minutes! Response time went from 40-60 seconds to 3-5 seconds!

**Comment by drnic** [comment]
Thanks, looks like I hadn't cut a gem release since generator had been added.

I've now released v0.3.1 with the generator.
